CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## v1.1.0
+### 2024-09-10
+
+This version of HOSS provides support for products without CF compliance like SMAP L3
+Methods added to get dimension scales from coordinate attributes and grid mapping with overrides
+in the json file
```

Comment:
As a rule usually you put the 👍 emoji after you've made the change as a alert to me that you saw my comment, agreed with it and changed it.  When I see the emoji I expect to be able to look at the change and then resolve my comment.

Reply:
But since it looks like you've thumbed up a bunch of things, you can add a comment with the githash as a way to alert the commenter that you have addressed the issue, when you get there.

Reply:
Changes are in  commit [756f7c0](https://github.com/nasa/harmony-opendap-subsetter/pull/15/commits/756f7c05d69fc135b5fc39825f5d7e1dde5daada)

Reply:
New comments added. will resolve this